src/answers-headless.ts

```diff
@@ -36,8 +36,8 @@ export default class AnswersHeadless {
     private httpManager: HttpManager,
   ) {}
 
-  setQuery(query: string): void {
-    this.stateManager.dispatchEvent('query/set', query);
+  setQueryInput(input: string): void {
```

Comment:
I updated this to be consistent with our convention of naming the setters 'setQuery + prop', however I'm not the biggest fan of this naming so I'm considering keeping `setQuery`. I'm not sure if that would make it unclear where the corresponding data is in the state. Let me know your thoughts

Reply:
I like the idea of keeping it `setQuery`. The term `setQueryInput` has a front-end connotation to me. It sounds very much like I'm setting the value of the `<input>` tag for a search bar.

Reply:
reverted to `setQuery`